src/main/java/org/takes/facets/auth/PsBasic.java

```diff
@@ -49,7 +50,7 @@
  * <p>The class is immutable and thread-safe.
  *
  * @author Endrigo Antonini (teamed@endrigo.com.br)
- * @version $Id$
+ * @version $Id: 5913e70e4877ecdd3f0cb85821dd8593f48a166a $
```

Comment:
@dalifreire remove the unneeded string at the end, this field should be `@version $Id$` only